Bottom: de454d0fbfa701f3e099d6d0b9e0d65c98242289
Top:    f32746daaeef8b0f439a36a0e499688b2a31fde5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 18:19:29 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
